magento/magento-cloud-patches
# MCLOUD-6588: Remove amazon patches from magento2-base component

### Description


### Fixed Issues (if relevant)

1. patch for  amzn/amazon-pay-module exist for magento2-base
2. https://github.com/magento/magento-cloud-patches/issues/51

### Manual testing scenarios

1. Check that amazon patches are applyed for next versions: 
-  Magento 2.2.6. - patch MAGECLOUD-2850__fix_amazon_payment_module__2.2.6.patch 
-  Magento 2.3.5 - patch BUNDLE-2554__set_payment_info_bug_fix__3.4.1.patch 
2. after removing amzn/amazon-pay-module or amzn/amazon-pay-and-login-magento-2-module patches should not be applied and deploy finishes without error

### Release notes

For user-facing changes, add a meaningful release note. For examples, see [Magento Cloud Patches release notes](https://devdocs.magento.com/cloud/release-notes/mcp-release-notes.html).

### Associated documentation updates

Add link to Magento DevDocs PR or Issue, if needed.

### Contribution checklist
